Runtimes/Core/Readme.md

```diff
@@ -0,0 +1,92 @@
+# SwiftCore
+
+SwiftCore contains core libraries that sit below the platform overlays.
+These include the standard library and associated runtimes, SwiftOnoneSupport,
+and the concurrency runtimes.
+
+## Build Instructions
+
+> [!IMPORTANT]
+> The standard library requires that it is built with a Swift compiler that is
+> at least as new as the standard library sources. You will likely need to
+> build the compiler as you would normally.
+> In these instructions, `<swiftc>` is the path to your just-built Swift
+> compiler.
+
+First create a build directory in the `Core/` directory and navigate to it.
+
+```sh
+mkdir build && cd build
+```
+
+Configure a simple default build. This results in a static build of the standard
+library.
+
+```sh
+cmake -G 'Ninja' -DCMAKE_Swift_COMPILER=<swiftc> ../`
+ninja
+```
```

Comment:
```suggestion
Configure a simple default build, which builds a static version of the standard library.

```sh
cmake -B build -G Ninja -S . -D CMAKE_Swift_COMPILER=<swiftc>
ninja -C build
```
```

I'd like to throw in some bike shedding for fun. I tend to prefer the build directory to be named `out`.